src/components/text-editor/examples/text-editor-basic.tsx

```diff
@@ -0,0 +1,35 @@
+import { LimelTextEditorCustomEvent } from '@limetech/lime-elements';
+import { Component, h, State } from '@stencil/core';
+/**
+ * A basic example
+ *
+ * Try typing and editing text, or copy & paste in some rendered HTML code
+ * from your browser into the editor to see how it is rendered and what you get
+ * as an output value.
+ */
+@Component({
+    tag: 'limel-example-text-editor-basic',
+    shadow: true,
+})
+export class BasicTextEditorBasicExample {
+    @State()
+    private text: { html: string } = { html: '' };
+
+    public render() {
+        return [
+            <limel-text-editor onChange={this.handleChange} />,
+            <hr />,
+            <h2>Preview using limel-markdown:</h2>,
+            <limel-markdown value={this.text.html} />,
+            <hr />,
```

Comment:
We're importing the markdown styles into this component.
So this section shouldn't really be necessary.
We can keep it for now while we're testing things. 
However when we publish the component, we don't need to have this in the docs.

Reply:
⚡